### PR TITLE
Prepare for release 1.2.2

### PR DESCRIPTION
**Pull request by Leon-Bichmann**
This PR adds a number of new R dependencies, to make it possible to use a speed increased version of DIAlignR for chromatogram alignment.

As part of this extracted chromatograms will be compressed in the sqMass instead of the mzML file format.

Moreover the AWS full test should be fixed.

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
 - [x] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
 - [x] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/diaproteomics/tree/master/.github/CONTRIBUTING.md)
 - [x] If necessary, also make a PR on the nf-core/diaproteomics _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint .`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by timosachsenberg — APPROVED** [review]

File: environment.yml [hunk 1]
```diff
@@ -16,6 +16,12 @@ dependencies:
   - bioconda::pyprophet=2.1.4
   - conda-forge::networkx=2.4
   - bioconda::easypqp=0.1.8
+  - conda-forge::r-rmsnumpress=1.0.1
```
Comment (shubham1637) [inline environment.yml:19]: Should the versions be ` >= ` instead of `=` ?

**Comment by shubham1637** [comment]
Hi Leon,  I would suggest using latest version:
`RUN Rscript -e 'remotes::install_github("shubham1637/DIAlignR@81befa7", dependencies=FALSE)'`
Set `params[["level"]] <- "Protein"` for Protein level FDR

**Comment by Leon-Bichmann** [comment]
> Hi Leon, I would suggest using latest version:
> `RUN Rscript -e 'remotes::install_github("shubham1637/DIAlignR@81befa7", dependencies=FALSE)'`
> Set `params[["level"]] <- "Protein"` for Protein level FDR

Alright I updated to the latest commit on master and params$level is set to "Protein" by default.

File: environment.yml (hunk 1 above)
Comment (Leon-Bichmann) [reply environment.yml:19]: I think it is better in this case to fix it to the specific version.

Comment (ewels) [reply environment.yml:19]: Definitely, as we want the pipeline to use exactly the same versions if running in the future for reproducibility. In fact I think that the nf-core linting will fail if you don't use `=`.

**Review by ewels — APPROVED** [review]
LGTM 👍🏻

**Comment by Leon-Bichmann** [comment]
I also added another low memory process option to the newly introduced FileCacher tool, as on a larger test dataset the memory consumption got too large.

**Pull request merged**